x-pack/test_serverless/functional/services/svl_search_navigation.ts

```diff
@@ -47,5 +47,15 @@ export function SvlSearchNavigationServiceProvider({
       });
       await testSubjects.existOrFail('searchIndicesDetailsPage', { timeout: 2000 });
     },
+    async navigateToInferenceManagementPage(expectRedirect: boolean = false) {
+      await retry.tryForTime(60 * 1000, async () => {
+        await PageObjects.common.navigateToApp('searchInferenceEndpoints', {
```

Comment:
There is no need to wrap `PageObjects.common.navigateToApp` with retry because it is already has retry inside with 2 times `timeouts.try` (4 minutes total)

https://github.com/elastic/kibana/blob/main/test/functional/page_objects/common_page.ts#L263-L373